HZ: 50
DT: 0.02

MAX_ROLL_ANGLE: 0.12217304763   # 7 deg
MAX_PITCH_ANGLE: 0.17453292519  # 10 deg

H: 0.1435
L: 0.30